grpclog/grpclog.go

```diff
@@ -30,9 +30,24 @@ import "os"
 
 var logger = newLoggerV2()
 
+// Verbose is a typed boolean that implements Infof.
+type Verbose bool
+
+// Infof is equivalent to the global Infof, guarded by the verbose value.
+func (v Verbose) Infof(format string, args ...interface{}) {
```

Comment:
Do you also want to provide bare `Info`?

Reply:
Added both `Info` and `Infoln`.

I was thinking `Infof` should work for all the cases.